tasgrid/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+########################################################################
+# tasgrid command line tool
+########################################################################
+
+########################################################################
+# add the tasgrid executable
+########################################################################
+add_executable(Tasmanian_tasgrid tasgrid_main.cpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/TasmanianSparseGrid.hpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestCLICommon.hpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestExternalTests.hpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestExternalTests.cpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestTestHelpers.hpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestTestFunctions.hpp
+                                 ${CMAKE_CURRENT_SOURCE_DIR}/../SparseGrids/gridtestTestFunctions.cpp
+                                 tasgridWrapper.hpp
+                                 tasgridWrapper.cpp)
+
+target_link_libraries(Tasmanian_tasgrid Tasmanian_libsparsegrid
+                                        Tasmanian_libdream
+                                        Tasmanian_addons)
+                                        
+set_target_properties(Tasmanian_tasgrid PROPERTIES OUTPUT_NAME "tasgrid" CXX_EXTENSIONS OFF)
+Tasmanian_rpath_target(TARGET Tasmanian_tasgrid USE_CURRENT)
```

Comment:
You wan to say `Tasmanian_rpath_target(TARGET Tasmanian_tasgrid COMPONENTS SparseGrid)`

The `USE_CURRENT` doesn't do anything since there are no libraries build in this folder. You need the SparseGrid library.

Reply:
Changed. I also added DREAM and Addons.

Reply:
Gotcha. Changed in the latest push.